### ISSUE TITLE
Improve performance by adding explicit height and width to image elements

### ISSUE THREAD

**Issue by JennaySDavis** [GSA/889-tool#146]
When testing https://github.com/GSA/889-tool/issues/123, I found the following performance issue of adding explicit height and width to image elements to increase the performance on the 889 tool.  The 889 tool meets GSA performance standards, but this would enhance the performance on the screen when loaded. I've attached the images without explicit height and width. 

![Image](https://github.com/GSA/889-tool/assets/149595240/fbf7e6d6-d54e-4baf-9869-805e200622c8)

**Comment by JennaySDavis** [comment]
**#146 Acceptance Criteria**
 

| Pass/Fail | Description |
| --- | --- | 
|  |Improve performance by adding explicit height and width to image elements|


**Comments/Additional Notes**
N/A

**ADA Compliance (Automated scan via Chrome Lighthouse)**

| Criteria | Score |
| --- | --- | 
| Performance | |
| Accessibility | |
| Best Practices |  |


- JSD

**Comment by johnbeallgsa** [comment]
@JennaySDavis, I see the 93 for best practices on a few tickets. Do we have another ticket that speaks about that?

**Comment by JennaySDavis** [comment]
@johnbeallgsa Yes, I have created ticket #162 for the developers to review the issues identified in the Lighthouse report.

**Comment by johnbeallgsa** [comment]
Thank you for showing this in Demo. Moving to Done.
